kyuubi-hive-beeline/src/main/java/org/apache/hive/beeline/BeeLine.java

```diff
@@ -1012,6 +1010,14 @@ private String getDefaultConnectionUrl(CommandLine cl) throws BeelineConfFilePar
       userConnectionProperties = userHS2ConnFileParser.getConnectionProperties();
     }
 
+    if (kyuubiConfFileParser.configExists()) {
+      // get the connection properties from user specific config file
+      Properties properties = kyuubiConfFileParser.getConnectionProperties();
+      for (String key : properties.stringPropertyNames()) {
+        userConnectionProperties.setProperty(key, properties.getProperty(key));
```

Comment:
We should merge `HIVE_CONF_PROPERTY_KEY` instead of overwriting.

Reply:
done.